Chap_API_Event.tex

```diff
@@ -43,7 +43,10 @@ \section{Notification and Management}
 %
 \end{itemize}
 
-Users can specify the callback order of a handler within its category at the time of registration. Ordering can be specified by providing the relevant event handler names, if the user specified an event handler name when registering the corresponding event. Thus, users can specify that a given handler be executed before or after another handler should both handlers appear in an event chain (the ordering is ignored if the other handler isn't included). Note that ordering does not imply immediate relationships. For example, multiple handlers registered to be serviced after event handler \textit{A} will all be executed after \textit{A}, but are not guaranteed to be executed in any particular order amongst themselves.
+Users can specify the callback order of a handler within its category at the time of registration. 
+Users can specify that a given handler be executed before or after another target handler should both handlers appear in an event chain (the ordering is ignored if the other handler isn't included). 
+The ordering is dictated by providing the event handler name of the target.  The name must have been assigned the target handler was registered.
```

Comment:
```suggestion
The ordering is dictated by providing the target event handler name.  The name must have been assigned when the target handler was registered.
```

Reply:
The wording on the first part was intentional, though your suggestion may still be valid.  The reason we said it the way we did "event handler name of the target" was because event handlers may not have names, so we wanted it to be especially clear that you are providing the name of the target (which you would need to define) as opposed to "target event handler name" which would more easily make the reader believe that such a name exists.  Maybe your approach is clear enough though and certainly reads better, so we will discuss next week.